redbot/core/drivers/red_mongo.py

```diff
@@ -49,6 +49,10 @@ def __init__(self, cog_name, identifier, **kwargs):
         if _conn is None:
             _initialize(**kwargs)
 
+    async def has_valid_connection(self) -> bool:
+        # Maybe fix this?
```

Comment:
Probably doesn't need fixing. With mongo selected, if there isn't a valid connection, the bot hangs prior to connect waiting for a token it wont get from mongo.